keeper-gradle-plugin/src/main/java/com/slack/keeper/KeeperPlugin.kt

```diff
@@ -122,11 +122,15 @@ public class KeeperPlugin : Plugin<Project> {
             property("android.enableR8")?.toString()?.toBoolean() != false
         if (r8Enabled) {
           appExtension.onApplicableVariants(project, extension) { testVariant, appVariant ->
+            val appR8Task = "minify${appVariant.name.capitalize(Locale.US)}WithR8"
+            val androidTestL8Task = "l8DexDesugarLib${testVariant.name.capitalize(Locale.US)}"
+            logger.debug("KEEPER: Patching $androidTestL8Task with ")
```

Comment:
nit: remove this or add `appR8Task` to it?

Reply:
I think this was left over from some debugging... let me look into it more 😓